vivisect/qt/remote.py

```diff
@@ -119,17 +120,18 @@ def getServer(self):
         return wsserver
 
 
-@vq_main.idlethread
 def openServerAndWorkspace(vw, parent=None):
     dia = VivConnectServerDialog(vw, parent=parent)
     host = dia.getServer()
     if host is None:
         return
+    if ':' in host:
+        host, port = host.split(':')
```

Comment:
shouldn't the port be pass to the `connServerAndWorkspace` call?

Reply:
heh. Yes. Yes it should. Thanks for that.